sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/DefaultScheduler.java

```diff
@@ -2,7 +2,7 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.protobuf.TextFormat;
-import com.mesosphere.sdk.scheduler.recovery.RecoveryPlanManagerFactory;
+import com.mesosphere.sdk.scheduler.recovery.*;
```

Comment:
Yeah, it's IntelliJ.  All the imports are managed by it.  I haven't manually imported anything...ever.  In any case I can do whatever anyone likes.